app/components/work_packages/progress/base_modal_component.rb

```diff
@@ -40,21 +40,32 @@ class BaseModalComponent < ApplicationComponent
         "remainingTime" => :remaining_hours,
         "work_package[remaining_hours]" => :remaining_hours,
         "work_package[status_id]" => :status_id,
-        "statusId" => :status_id
+        "statusId" => :status_id,
+        "work_package[done_ratio]" => :done_ratio,
+        "percentageDone" => :done_ratio,
+        "" => :no_field
       }.freeze
 
       include ApplicationHelper
       include Turbo::FramesHelper
       include OpPrimer::ComponentHelpers
       include OpenProject::StaticRouting::UrlHelpers
 
-      attr_reader :work_package, :mode, :focused_field, :touched_field_map
+      attr_reader :work_package,
+                  :mode,
+                  :focused_field,
+                  :format_durations,
```

Comment:
It looks like `format_durations` is now always true, and so is not needed anymore and can be removed. Am I right?

Reply:
I removed it